### PR TITLE
feat: add tailwind

### PR DESCRIPTION
**Pull request by acd02**
related to #167

### CODE REVIEW

**Review by arnau-rius — APPROVED** [review]
👏🏽

File: package.json [hunk 1]
```diff
@@ -50,10 +51,14 @@
     "eslint-plugin-simple-import-sort": "10.0.0",
     "eslint-plugin-standard": "5.0.0",
     "eslint-plugin-storybook": "^0.6.10",
+    "eslint-plugin-tailwindcss": "3.8.3",
+    "postcss": "8.4.21",
     "prettier": "2.8.3",
+    "prettier-plugin-tailwindcss": "0.2.2",
```
[the remark this answers is not in the document]
Comment (acd02) [reply package.json:57]: Nope, Prettier picks it up automatically from the package.json 👍

[the remark this answers is not in the document]
Comment (arnau-rius) [reply package.json:57]: ohh nice

[the remark this answers is not in the document]
Comment (arnau-rius) [reply package.json:57]: ohh nice

**Pull request merged**